src/Microsoft.Health.Dicom.Tests.Common/Comparers/DicomItemCollectionEqualityComparer.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Health.Dicom.Tests.Common.Comparers
     public class DicomItemCollectionEqualityComparer : IEqualityComparer<IEnumerable<DicomItem>>
     {
         public DicomItemCollectionEqualityComparer()
-            : this(new DicomTag[0])
+            : this(new DicomTag[] { DicomTag.ImplementationVersionName })
```

Comment:
I'm wondering why we change the default value? would it make more sense to put that into the caller?